package.json

```diff
@@ -2,41 +2,34 @@
   "name": "tyranid",
   "version": "0.2.2",
   "description": "Metadata Library to faciliate working generically with data (MongoDB, ODM)",
-  "main": "dist/tyranid.js",
-  "jsnext:main": "src/tyranid.js",
+  "main": "dist/src/tyranid.js",
   "typings": "src/tyranid.d.ts",
```

Comment:
actually it looks like you can't output declaration files with `allowJS`, so we can change it once the full project is converted.